src/module.c

```diff
@@ -845,6 +871,9 @@ int TSDB_get(RedisModuleCtx *ctx, RedisModuleString **argv, int argc) {
 
     ReplyWithSeriesLastDatapoint(ctx, series);
     RedisModule_CloseKey(key);
+
```

Comment:
@gkorland why notify on get?

Reply:
right... over motivation.... I'll remove it